ctfpad/templates/ctfpad/dashboard/dashboard.html

```diff
@@ -10,6 +10,16 @@
 
 {% include 'snippets/messages.html' %}
 
+<script>
+fetch("{{CODIMD_URL}}/login", {
+  method: 'POST',
+  mode: 'no-cors',
+  cache: 'no-cache',
+  credentials: 'include',
+  headers: { 'Content-Type': 'application/x-www-form-urlencoded'},
+  body: "email={{request.user}}%40localhost.localdomain&password=hedgedoc"});
```

Comment:
why not use `request.user.email` ?

Reply:
Because if I register the `lanjelot` on ctfpad using `hahanfiwhothisis@gmail.com` and we use `request.user.email` instead of `request.user` then hedgedoc will use `hahanfiwhothisis` as the username and we'll have no idea that it is actually lanjelot (hedgedoc uses everything before @ as the username)